src/yaml/ceafus-88/27490-frex-ceafus-stop-sign-pack.yaml

```diff
@@ -0,0 +1,40 @@
+group: ceafus-88
+name: stop-sign-pack
+version: "1.0"
+subfolder: 700-transit
+info:
+  summary: Frex_Ceafus Stop Sign Pack
+  description: |-
+    Hello, I am proud to bring you a stop sign pack made by myself, but this is more than just any regular stop sign pack, but a pack that includes stop signs from other countries, including,
+
+    -   China  
+        
+    -   U.S.A  
+        
+    -   UK  
+        
+    -   Japan  
+        
+    -   and South America  
+        
+
+    Please only choose one of these to put in your plugin's folder, and enjoy your favorite stop signs!
```

Comment:
The description says choose only one. Does this mean it they override eachother? In that case, I would add variants for every country.